lib/MT/DataAPI/Callback/TemplateMap.pm

```diff
@@ -85,7 +70,10 @@ sub _retrieve_archive_types {
         {
 
             # only include if it is NOT an entry-based archive type
-            next if $archiver->entry_based;
+            next
+                if $archiver->entry_based
+                && $archiver->contenttype_based
+                && $archiver->contenttype_group_based;
```

Comment:
@miniuchi  ```! $archiver->contenttype_(group_)?based``` ?

Reply:
@charsbar Thanks, but probably `next if $archiver->entry_based || $archiver->contenttype_based || $archiver->contenttype_group_based`

Reply:
@miniuchi ah, seems so.

Reply:
@miniuchi BTW is this really required (when $obj_type is archive/author/category)?